### PR TITLE
:ghost: Add effort column to table and drawer

### PR DESCRIPTION
**Pull request by ibolton336**
- Adds story points/effort field
Closes [Adding Efforts Column in Application Inventory Analysis](https://issues.redhat.com/browse/MTA-1535)

### CODE REVIEW

**Review by gitdallas — APPROVED** [review]

File: client/src/app/pages/applications/applications-table-analyze/applications-table-analyze.tsx [hunk 1]
```diff
@@ -636,6 +645,13 @@ export const ApplicationsTableAnalyze: React.FC = () => {
                       <TagIcon />
                       {application.tags ? application.tags.length : 0}
                     </Td>
+                    <Td
+                      width={10}
+                      modifier="truncate"
+                      {...getTdProps({ columnKey: "effort" })}
+                    >
+                      {application.effort ? application.effort : "-"}
```
Comment (sjd78) [inline client/src/app/pages/applications/applications-table-analyze/applications-table-analyze.tsx:653]:
```suggestion
                      {application?.effort ?? "-"}
```
...the optional chaining with nullish coalescing way

File: client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx [hunk 2]
```diff
@@ -129,6 +129,12 @@ export const ApplicationDetailDrawer: React.FC<
                 ? application.migrationWave.name
                 : t("terms.unassigned")}
             </Text>
+            <Title headingLevel="h3" size="md">
+              {t("terms.effort")}
+            </Title>
+            <Text component="small">
+              {application?.effort ? application.effort : t("terms.unassigned")}
```
Comment (sjd78) [inline client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx:136]:
```suggestion
              {application?.effort ?? application.effort : t("terms.unassigned")}
```
...the optional chaining with nullish coalescing way

**Review by sjd78 — APPROVED** [review]
LGTM with 2 non-blocking suggestion

File: client/src/app/pages/applications/applications-table-analyze/applications-table-analyze.tsx (hunk 1 above)
Comment (ibolton336) [reply client/src/app/pages/applications/applications-table-analyze/applications-table-analyze.tsx:653]: Very nice.

File: client/src/app/pages/applications/applications-table-analyze/applications-table-analyze.tsx [hunk 3]
```diff
@@ -636,6 +645,13 @@ export const ApplicationsTableAnalyze: React.FC = () => {
                       <TagIcon />
                       {application.tags ? application.tags.length : 0}
                     </Td>
+                    <Td
+                      width={10}
+                      modifier="truncate"
+                      {...getTdProps({ columnKey: "effort" })}
+                    >
+                      {application?.effort && "-"}
```
[the remark this answers is not in the document]
Comment (ibolton336) [reply client/src/app/pages/applications/applications-table-analyze/applications-table-analyze.tsx:653]: Oh crap. Fixing.

**Review by sjd78 — APPROVED** [review]
LGTM!
:smile:

**Pull request merged**